src/mono/wasm/host/WebServer.cs

```diff
@@ -21,6 +17,7 @@ public class WebServer
     internal static async Task<(ServerURLs, IWebHost)> StartAsync(WebServerOptions options, ILogger logger, CancellationToken token)
     {
         string[]? urls = new string[] { $"http://127.0.0.1:{options.Port}", "https://127.0.0.1:0" };
```

Comment:
would `0.0.0.0:0` IP work better ? just asking

Reply:
`0.0.0.0` would mean "listen on all the ip addresses for this machine", so it would be reachable over the local network, for example.

Reply:
What does the ASP.NET do ? Is it also only 127.0.0.1 ?
What if we only have IPv6 ? Shall we use `localhost` instead ?

Reply:
IIRC, they use 127.0.0.1 too. I imagine for IPv6 the user would have to specify the host.

There is https://learn.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.server.kestrel.core.kestrelserveroptions.listenlocalhost?view=aspnetcore-6.0#microsoft-aspnetcore-server-kestrel-core-kestrelserveroptions-listenlocalhost(system-int32) but that doesn't support dynamic ports either ;)
For 7.0, I think this is enough.

Reply:
okay